File: Em Python/2547.py
while 1:
    try:
        cont = 0
        n, amin, amax = map(int,input().split())
        for i in range(0,n):
            a = int(input())
            if (a >= amin) and (a <= amax):
                cont += 1
        print(cont)
    except EOFError:
        break

File: Em C/2547.c
#include <stdio.h>

int main(int argc, char** argv)
{
	int n,amin,amax,a;
	
	while(scanf("%d %d %d",&n,&amin,&amax) != EOF)
	{
		int cont = 0;
		while(n--)
		{
			scanf("%d",&a);
			if(a >= amin && a <= amax){
				cont+=1;
			}
		}
		printf("%d\n",cont);
	}
	return 0;
}